e2e/virtualmachinevolume.go

```diff
@@ -12,7 +13,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"kubevirt-image-service/pkg/apis"
 	hc "kubevirt-image-service/pkg/apis/hypercloud/v1alpha1"
-	volume "kubevirt-image-service/pkg/controller/virtualmachinevolume"
+	v "kubevirt-image-service/pkg/controller/virtualmachinevolume"
```

Comment:
The package name is used for readability. see this link https://rakyll.org/style-packages/